docs/schemaform/form-config.md

```diff
@@ -25,6 +25,9 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
   // Schema definitions that can be referenced on any page. These are added to each page's schema
   // in the reducer code, so that you don't have to put all of the common fields in the definitions
   // property in each page schema.
+  // In a schema's properties, there are references: "homePhone": { "$ref": "#/definitions/phone" }
+  // In the config file, the definition for phone would then need to be added into
+  // defaultDefinitions for it to be parsed correctly and added to homePhone
```

Comment:
It might make sense to add a `definitions` property in the `schema` object in here and put this description there. In JSON Schema, the usual way of specifying definitions for fields using `$ref` is in a `definitions` property at the root of the schema. The point of `defaultDefinitions` is that we will add the definition in it to each page's schema automatically, instead of forcing a developer to add them to each page's schema individually.